onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -171,7 +189,7 @@ Status QkvToContext(
 
   // compute Q*K' (as K'*Q), scaled by 1/sqrt(H) and store in scratch1: BxNxSxS*
   // Q: BxNxSxH, K (present_k): BxNxS*xH, Q*K': BxNxSxS*
-  const float rsqrt_head_size = 1.f / sqrt(static_cast<float>(head_size));
+  const float rsqrt_head_size = 1.f / sqrt(static_cast<float>(qkv_head_size[0]));
```

Comment:
An explanation that qk head size is used here instead of v head size.

Reply:
Modified the comment to specify H_qk is used, not H_v - not sure if further clarification is needed